vaadin-platform-test/src/test/java/com/vaadin/platform/test/ChromeComponentsIT.java

```diff
@@ -83,8 +83,13 @@ protected String getTestPath() {
     }
 
     @Before
-    public void load(){
-        $(NotificationElement.class).waitForFirst();
+    public void load() {
+        if (getDriver().getPageSource().contains("<div class=\"message\">Building front-end development bundle</div>")) {
+            // Wait for the development bundle to be loaded
+            $(NotificationElement.class).waitForFirst(180);
```

Comment:
I feel that ff the problem is that we need for the bundle to be compiled, probably this fix should be in the testbench instead